### ISSUE TITLE
[Bug]: Testcases get stucked for a very long time 

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#21358]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:v2.2.0
- Deployment mode(standalone or cluster):cluster
- MQ type(rocksmq, pulsar or kafka): kafka    
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus==2.2.1.dev5
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

![image](https://user-images.githubusercontent.com/12268675/209266078-1853d5fc-80a6-43cb-9da5-2ee31eaca340.png)

![image](https://user-images.githubusercontent.com/12268675/209265708-c9c48d79-93c6-4e00-8914-91bbac469cea.png)

2022-12-22T12:35:58.125Z --> 2022-12-22T14:02:58.501Z
There is a big time gap for two test cases finished time

### Expected Behavior

![image](https://user-images.githubusercontent.com/12268675/209266026-61964018-39ff-4b73-b78b-740df783dde2.png)

### Milvus Log

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test_kafka_for_release_cron/detail/deploy_test_kafka_for_release_cron/127/pipeline
log:
[artifacts-kafka-standalone-upgrade-127-server-second-deployment-logs.tar.gz](https://github.com/milvus-io/milvus/files/10291886/artifacts-kafka-standalone-upgrade-127-server-second-deployment-logs.tar.gz)

[artifacts-kafka-standalone-upgrade-127-pytest-logs.tar.gz](https://github.com/milvus-io/milvus/files/10291887/artifacts-kafka-standalone-upgrade-127-pytest-logs.tar.gz)

**Comment by zhuwenxing** [comment]
image version: v2.2.0
![image](https://user-images.githubusercontent.com/12268675/209522125-f5ef7e4b-6a2f-4ee0-8886-df21367d740d.png)

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test_kafka_cron/detail/deploy_test_kafka_cron/131/pipeline/134

log:

[artifacts-kafka-cluster-upgrade-131-server-second-deployment-logs.tar.gz](https://github.com/milvus-io/milvus/files/10302381/artifacts-kafka-cluster-upgrade-131-server-second-deployment-logs.tar.gz)

[artifacts-kafka-cluster-upgrade-131-pytest-logs.tar.gz](https://github.com/milvus-io/milvus/files/10302383/artifacts-kafka-cluster-upgrade-131-pytest-logs.tar.gz)

**Comment by zhuwenxing** [comment]
image tag: v2.2.0 

failed job： https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test_kafka_cron/detail/deploy_test_kafka_cron/178/pipeline/76

log：
[Uploading artifacts-kafka-cluster-upgrade-178-pytest-logs.tar.gz…]()
[artifacts-kafka-cluster-upgrade-178-server-second-deployment-logs.tar.gz](https://github.com/milvus-io/milvus/files/10357040/artifacts-kafka-cluster-upgrade-178-server-second-deployment-logs.tar.gz)

**Comment by yanliang567** [comment]
@jaime0815 any updates?

**Comment by jaime0815** [comment]
related to https://github.com/milvus-io/milvus/issues/20918,  it had been fixed on branch 2.2

**Comment by zhuwenxing** [comment]
> related to #20918, it had been fixed on branch 2.2

But the stuck case happens in the first deployment, which's image is v2.2

**Comment by zhuwenxing** [comment]
When the old image tag changed to v2.2.2, it works well
